scripts/synapse_port_db

```diff
@@ -820,14 +820,12 @@ class Porter(object):
                     "ALTER SEQUENCE events_stream_seq RESTART WITH %s", (next_id,)
                 )
 
-            txn.execute("SELECT -MIN(stream_ordering) FROM events")
+            txn.execute("SELECT GREATEST(-MIN(stream_ordering), 1) FROM events")
             curr_id = txn.fetchone()[0]
-            if curr_id:
-                next_id = curr_id + 1
-                txn.execute(
-                    "ALTER SEQUENCE events_backfill_stream_seq RESTART WITH %s",
-                    (next_id,),
-                )
+            next_id = curr_id + 1
```

Comment:
sooo.. if there are no backfilled events, the SELECT will return 1, so we'll set the RESTART value to 2. I mean, it's not the end of the world, but it doesn't sound right.

Reply:
I mean, it really doesn't matter? I can change it if you want, but we do the exact same thing from a fresh DB too

Reply:
we do? fair enough. Thought you might change the GREATEST to 0.  I don't feel strongly.